contrib/brl/bbas/bdgl/bdgl_curve_algs.h

```diff
@@ -21,9 +21,13 @@
 class bdgl_curve_algs
 {
  public:
-  static bdgl_EXPORT_DATA const double tol;
-  static bdgl_EXPORT_DATA const double max_edgel_sep;
-  static bdgl_EXPORT_DATA const double synthetic;
+  static bdgl_EXPORT_DATA constexpr double tol = 1e-6;
```

Comment:
This was 1e-16 and now is 1e-6.  I think that it may have been a typo.

Reply:
Good catch!  Fixed now.